packages/i18n/src/locales/en_US.ts

```diff
@@ -591,6 +599,41 @@ const translations: Catalog = {
           }
         }
       },
+      ChangeStreamCursor: {
+        iteration: {
+          'no-cursor': 'no cursor',
+          'type-it-for-more': 'Type "it" for more'
+        },
+        help: {
+          description: 'Change Stream Cursor',
+          attributes: {
+            close: {
+              description: 'Instructs the server to close a cursor and free associated server resources.',
+            },
+            isClosed: {
+              description: 'Returns true if the cursor is closed'
+            },
+            itcount: {
+              description: 'Returns the number of documents in the current batch. NOTE: this method exhausts the cursor batch'
+            },
+            getResumeToken: {
+              description: 'Returns the ResumeToken of the change stream'
+            },
+            hasNext: {
+              description: 'WARNING: on change streams this method will block unless the cursor is closed. Use tryNext to check if there are any documents in the batch. This is a breaking change'
+            },
+            next: {
+              description: 'WARNING: on change streams this method will block unless the cursor is closed. Use tryNext to get the next document in the batch. This is a breaking change'
+            },
+            tryNext: {
+              description: 'If there is a document in the change stream, it will be returned. Otherwise returns null.'
```

Comment:
When you call `tryNext` it iterates the cursor, so you could implement `hasNext` in a way that would act the same but the side effect is that you can no longer get the document out at all, since if you then call `next` the cursor won't have the doc anymore that was returned by the first `tryNext`

Reply:
We could cache the document and return it from `next`/hide that as an implementation detail if we wanted to though, right? Not saynig we should, just wondering if that’s a possibility

Reply:
That's a good point. I think because there are so many other backwards breaking changes to the tailable cursors (for example, isExhausted no longer makes sense since the driver does not allow you call tryNext after the cursor has closed) that I think maybe a more dramatic break would be better so people don't end up making subtle mistakes. I asked Max this morning and he linked me to this comment by dan: https://docs.google.com/document/d/1tGB65zRhH3CKSlzg8YIpzCtShOpY8sWDDsgZJZL4wkI/edit?disco=AAAAHXcXDJ0. So I don't think backwards-breaking changes to this API are a big problem as long as we document well